fuel/src/test/kotlin/com/github/kittinunf/fuel/BaseTestCase.kt

```diff
@@ -2,6 +2,8 @@ package com.github.kittinunf.fuel
 
 abstract class BaseTestCase {
 
+
```

Comment:
**[ktlint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Needless blank line(s)

Reply:
Finally ... it works ...